catalog-rest-service/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -488,10 +541,6 @@
           "description": "No.of rows in the table. This is always executed on the whole table.",
           "type": "number"
         },
-        "columnNames": {
-          "description": "List of column names",
```

Comment:
@TeddyCr I am not sure if we used this for the column names metric

Reply:
Ya not sure if we need this though in the columnProfiler, we have getName() that supposed to be the column Name

Reply:
I think the only reason for the names to be here is to "document" the table's state during the execution of the profiler at a given ts. Kind of to decouple the profiler from having to research versioning etc.

IDK how valuable it is to keep copying that data though, so no issues on removing it

Reply:
we'll need to update `columnNameToExist` test as it relies on that. We'll need to fetch all the column names from somewhere else.

Reply:
@TeddyCr you can make a call to table.getColumns() to look up right.

Reply:
Yep make sense.